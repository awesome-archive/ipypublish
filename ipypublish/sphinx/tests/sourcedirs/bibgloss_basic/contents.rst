Welcome to IpySphinx extension tests documentation!
===================================================


:glsc:`term1`, :gls:`term2`, :gls:`acro1`

:glscpl:`term1`, :glspl:`term2`

.. rubric:: Glossary

.. bibglossary:: test_glossary.bib



