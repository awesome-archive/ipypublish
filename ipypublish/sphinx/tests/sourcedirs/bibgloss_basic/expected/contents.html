
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <title>Welcome to IpySphinx extension tests documentation! &#8212; Python  documentation</title>
    <link rel="stylesheet" href="_static/alabaster.css" type="text/css" />
    <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
    <script type="text/javascript">
      var DOCUMENTATION_OPTIONS = {
        URL_ROOT:    './',
        VERSION:     '',
        COLLAPSE_INDEX: false,
        FILE_SUFFIX: '.html',
        HAS_SOURCE:  true,
        SOURCELINK_SUFFIX: '.txt'
      };
    </script>
    <script type="text/javascript" src="_static/jquery.js"></script>
    <script type="text/javascript" src="_static/underscore.js"></script>
    <script type="text/javascript" src="_static/doctools.js"></script>
    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
   
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
  
  
  <meta name="viewport" content="width=device-width, initial-scale=0.9, maximum-scale=0.9" />

  </head>
  <body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          <div class="body" role="main">
            
  <div class="section" id="welcome-to-ipysphinx-extension-tests-documentation">
<h1>Welcome to IpySphinx extension tests documentation!<a class="headerlink" href="#welcome-to-ipysphinx-extension-tests-documentation" title="Permalink to this headline">Â¶</a></h1>
<p><a class="bibglossary bibgcapital reference internal" href="#term1" id="id1">Name</a>, <a class="bibglossary reference internal" href="#term2" id="id2">name2</a>, <a class="bibglossary reference internal" href="#acro1" id="id3">OTHER</a></p>
<p><a class="bibglossary bibgplural bibgcapital reference internal" href="#term1" id="id4">Names</a>, <a class="bibglossary bibgplural reference internal" href="#term2" id="id5">name2 plural</a></p>
<p class="rubric">Glossary</p>
<p id="bibtex-bibglossary-contents-0"><table class="bibglossary docutils citation" frame="void" id="term1" rules="none">
<colgroup><col class="label" /><col /></colgroup>
<tbody valign="top">
<tr><td class="label">[name]</td><td><em>(<a class="fn-backref" href="#id1">1</a>, <a class="fn-backref" href="#id4">2</a>)</em> the description which contains latex <span class="math">\(\frac{-23}{129}\)</span></td></tr>
</tbody>
</table>
<table class="bibglossary docutils citation" frame="void" id="term2" rules="none">
<colgroup><col class="label" /><col /></colgroup>
<tbody valign="top">
<tr><td class="label">[name2]</td><td><em>(<a class="fn-backref" href="#id2">1</a>, <a class="fn-backref" href="#id5">2</a>)</em> <p>the description which contains</p>
<p class="last">a second paragraph</p>
</td></tr>
</tbody>
</table>
<table class="bibglossary docutils citation" frame="void" id="acro1" rules="none">
<colgroup><col class="label" /><col /></colgroup>
<tbody valign="top">
<tr><td class="label"><a class="fn-backref" href="#id3">[OTHER]</a></td><td>Abbrev of other</td></tr>
</tbody>
</table>
</p>
</div>


          </div>
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
        <div class="sphinxsidebarwrapper"><div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="#">Documentation overview</a><ul>
  </ul></li>
</ul>
</div>
  <div role="note" aria-label="source link">
    <h3>This Page</h3>
    <ul class="this-page-menu">
      <li><a href="_sources/contents.rst.txt"
            rel="nofollow">Show Source</a></li>
    </ul>
   </div>
<div id="searchbox" style="display: none" role="search">
  <h3>Quick search</h3>
    <form class="search" action="search.html" method="get">
      <div><input type="text" name="q" /></div>
      <div><input type="submit" value="Go" /></div>
      <input type="hidden" name="check_keywords" value="yes" />
      <input type="hidden" name="area" value="default" />
    </form>
</div>
<script type="text/javascript">$('#searchbox').show(0);</script>
        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &copy;.
      
      |
      Powered by <a href="http://sphinx-doc.org/">Sphinx 1.6.3</a>
      &amp; <a href="https://github.com/bitprophet/alabaster">Alabaster 0.7.10</a>
      
      |
      <a href="_sources/contents.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>